src/index.html

```diff
@@ -12,6 +12,185 @@
   </head>
   <body>
     <h1>HTML Form</h1>
+
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="fieldset">
+        <legend class="legend">Personal information</legend>
+        <label class="form-field">
+        Surname:
+          <input
```

Comment:
Fix text indents everywhere
```suggestion
        <label class="form-field">
          Surname:
          <input
```